Okta-Application/okta-application-application.json

```diff
@@ -279,7 +293,9 @@
   ],
   "writeOnlyProperties": [
     "/properties/Credentials",
-    "/properties/RequestObjectSigningAlg"
+    "/properties/RequestObjectSigningAlg",
+    "/properties/Settings/App/Url",
```

Comment:
Why are these write only?

Reply:
The values added are not returned by the READ handler hence need to be added as writeOnlyProps to pass `test_read_input_output_negative_match` test.